content/en/checkout/guide.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Guide
-description: ""
+description: ''
```

Comment:
Can you revert these linter changes so we keep it consistent with the other files?